<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Peliculas</class>
 <widget class="QDialog" name="Peliculas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pelicula</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nombre"/>
       </item>
       <item>
        <widget class="QPushButton" name="Boton_buscar">
         <property name="text">
          <string>Buscar pelicula</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sinopsis</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Clasificacion</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_nombre_completo"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_genero"/>
       </item>
       <item row="1" column="3" rowspan="3">
        <widget class="QTextEdit" name="textEdit_sinopsis"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Genero</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre completo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_duracion"/>
       </item>
       <item row="7" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="Boton_agregar">
           <property name="text">
            <string>Agregar pelicula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Boton_modificar">
           <property name="text">
            <string>Modificar pelicula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Boton_eliimnar">
           <property name="text">
            <string>Eliminar pelicula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Boton_guardar">
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Duracion</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_clasificacion"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
